python/S3_VPC_ENDPOINT_ENABLED/S3_VPC_ENDPOINT_ENABLED_test.py

```diff
@@ -0,0 +1,335 @@
+                            "HostedZoneId": "Z7HUB22UULQXV"
+                        }
+                    ],
+                    "CreationTimestamp": "2018-08-15T14:29:07.000Z"
+                }
+            ]
+        }
+        EC2_CLIENT_MOCK.describe_vpc_endpoints = MagicMock(return_value=describevpcendpoints_return)
+        resp_expected = []
+        response = RULE.lambda_handler(build_lambda_scheduled_event(), {})
+        resp_expected.append(build_expected_response('NON_COMPLIANT', 'vpc-1234567', compliance_resource_type='AWS::EC2::VPC', annotation='There are no Amazon S3 VPC endpoints present in '+ vpc_id+'.'))
+        assert_successful_evaluation(self, response, resp_expected)
+
+    #Unit test for when Amazon VPC has a AWS S3 VPC present but is not in available state -- Gherkin Scenario 3
+    def test_scenario_3(self):
+        EC2_CLIENT_MOCK.reset_mock()
+        resp_expected = []
+        vpc_id = "vpc-1234567"
+        vpc_return = {'Vpcs':[{'VpcId':'vpc-1234567'}]}
+        EC2_CLIENT_MOCK.describe_vpcs = MagicMock(return_value=vpc_return)
+        describevpcendpoints_return = {
+            "VpcEndpoints": [
+                {
+                    "VpcEndpointId": "vpce-06e48c97f0bc5d501",
+                    "VpcEndpointType": "Gateway",
+                    "VpcId": "vpc-0056487b",
+                    "ServiceName": "com.amazonaws.us-east-1.s3",
+                    "State": "pending",
+                    "PolicyDocument": "{\"Version\":\"2008-10-17\",\"Statement\":[{\"Effect\":\"Allow\",\"Principal\":\"*\",\"Action\":\"*\",\"Resource\":\"*\"}]}",
+                    "RouteTableIds": [
+                        "rtb-0ba1ba4f40636b1fd"
+                    ],
+                    "SubnetIds": [],
+                    "Groups": [],
+                    "PrivateDnsEnabled": 'false',
+                    "NetworkInterfaceIds": [],
+                    "DnsEntries": [],
+                    "CreationTimestamp": "2019-03-29T18:25:47.000Z"
+                }
+            ]
+        }
+        EC2_CLIENT_MOCK.describe_vpc_endpoints = MagicMock(return_value=describevpcendpoints_return)
+        response = RULE.lambda_handler(build_lambda_scheduled_event(), {})
+        resp_expected.append(build_expected_response('NON_COMPLIANT', 'vpc-1234567', compliance_resource_type='AWS::EC2::VPC', annotation='The Amazon S3 VPC endpoint is not in Available state '+vpc_id+'.'))
+        assert_successful_evaluation(self, response, resp_expected)
+
+    #Unit test for VPC is present, and S3 endpoint is present in available state -- Gherkin Scenario 4
+    def test_scenario_4(self):
+        EC2_CLIENT_MOCK.reset_mock()
+        resp_expected = []
+        vpc_return = {'Vpcs':[{'VpcId':'vpc-1234567'}]}
+        EC2_CLIENT_MOCK.describe_vpcs = MagicMock(return_value=vpc_return)
+        describevpcendpoints_return = {
+            "VpcEndpoints": [
```

Comment:
Idem

Reply:
Please remove unnecessary data to improve readability

Reply:
done